docs/configuration.md

```diff
@@ -0,0 +1,109 @@
+
+
+## Relay
+
+The following table lists the available properties along with their default values for the [Relay package](/packages/relay/). 
+Unless you need to set a non-default value, it is recommended to only populate overridden properties in the custom `.env`.
+
+| Name                              | Default       | Description                                                   |
+|-----------------------------------|---------------|---------------------------------------------------------|
+| `CLIENT_TRANSPORT_SECURITY`       | "false"       | Flag to enable or disable TLS for both networks.   |
+| `CONSENSUS_MAX_EXECUTION_TIME`    | "15000"       | Maximum time in ms the SDK will wait when submitting a transaction/query before throwing a TIMEOUT error.   |
+| `DEFAULT_RATE_LIMIT`              | "200"         | default fallback rate limit, if no other is configured.   |
+| `ETH_CALL_CACHE_TTL`              | "200"         | Maximum time in ms to cache an eth_call response.   |
+| `ETH_CALL_CONSENSUS`              | "false"       | Flag to set if eth_call logic should first query the mirror node.   |
+| `ETH_GET_LOGS_BLOCK_RANGE_LIMIT`  | "1000"        | The maximum block number range to consider during an eth_getLogs call.   |
+| `FEE_HISTORY_MAX_RESULTS`         | "10"          | The maximum number of results to returns as part of `eth_feeHistory`.   |
+| `GAS_PRICE_TINY_BAR_BUFFER`       | "10000000000" | The additional buffer range to allow during a relay precheck of gas price. This supports slight fluctuations in network gasprice calculations.   |
+| `LIMIT_DURATION`                  | "60000"       | The maximum duration in ms applied to IP-method based rate limits.   |
+| `MIRROR_NODE_LIMIT_PARAM`         | "100"         | The mirror node custom limit value to be set on GET requests. This optimizes the flow to reduce the number of calls made to the mirror node by setting a limit larger than it's default limit.   |
+| `MIRROR_NODE_RETRIES`             | "3"           | The maximum number of retries on a GET request to the mirror node when an acceptable error code is returned.   |
+| `MIRROR_NODE_RETRY_DELAY`         | "250"         | The dealy in ms between retry requests.   |
+| `MIRROR_NODE_URL`                 | ""            | The Mirror Node API endpoint. Official endpoints are Previewnet (https://previewnet.mirrornode.hedera.com/v1/api), Testnet (https://testnet.mirrornode.hedera.com/v1/api), Mainnet (https://mainnet-public.mirrornode.hedera.com/v1/api). See [Mirror Node REST API](https://docs.hedera.com/hedera/sdks-and-apis/rest-api)   |
+| `SDK_REQUEST_TIMEOUT`             | "10000"       | The complete timeout for running the SDK `execute()` method. This controls the GRPC channel timeout cofig when querying with network nodes.   |
```

Comment:
Spelling mistake, "cofig" when it should be "config".

Reply:
Fixed